### ISSUE TITLE
Callout setInitialFocus does not work on Android

### ISSUE THREAD

**Issue by lutorre** [microsoft/fluentui#16284]
### Environment Information

- **Package version(s)**:  @fluentui/react 7.129.0
- **Browser and OS versions**: Android 11.0

### Describe the issue:

setInitialFocus property does not focus any focusable elements within a callout

### Please provide a reproduction of the issue in a codepen:
You can see this behavior by enabling a screen reader and using the "Callout" example from Fluent's documentation.

#### Actual behavior:

setInitialFocus property does not focus any focusable elements within a callout

#### Expected behavior:
setInitialFocus property set focus on the first focusable component within a Callout

**Comment by gouttierre** [comment]
@lutorre  -  Thanks for logging this issue.

@khmakoto  - Would you be able to determine if the setInitialFocus property does not focus any focusable elements within a callout is a regression or a behavior issue?
